src/DGtal/geometry/curves/CDynamicBidirectionalSegmentComputer.h

```diff
@@ -97,7 +98,7 @@ Description of \b concept '\b CDynamicBidirectionalSegmentComputer' <p>
     // ------------------------- Internals ------------------------------------
   private:
   }; // end of concept CDynamicBidirectionalSegmentComputer
-
+ }
```

Comment:
`// namespace concepts` (but OK, you will maybe not update all the 50 files again)